infra/lib/ci-cluster.ts

```diff
@@ -108,7 +108,7 @@ export class CICluster extends cdk.Construct {
       chart: 'external-dns',
       repository: 'https://charts.bitnami.com/bitnami',
       namespace: EXTERNAL_DNS_NAMESPACE,
-      version: '4.11.1',
+      version: '6.12.0',
```

Comment:
Nice! Looks like the version we were using was released 18months ago.. [1f0ff4b46a99155089b2bd19e9c0c2acad2069cc](https://github.com/bitnami/charts/commit/1f0ff4b46a99155089b2bd19e9c0c2acad2069cc)